test_scripts/WebEngine/WebEngineProjection/003_Check_WSS_connection_for_WebEngine_projection_app_in_NONE.lua

```diff
@@ -29,10 +29,9 @@ local appHMIType = "WEB_VIEW"
 
 --[[ General configuration parameters ]]
 config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
-config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
-config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
 config.defaultMobileAdapterType = "WSS"
```

Comment:
@aderiabin BTW  I left it as it is because if I delete this line, the script will abort

Reply:
@LitvinenkoIra Ok, let's leave this line for a while.
Maybe this decision will be changed on final review.